src/analytics/AnalyticsPage.react.js

```diff
@@ -3,10 +3,19 @@
  * @author Daniel Tat
  */
 
+import Container from '@mui/material/Container';
 import * as React from 'react';
 
+import AnalyticsHeader from './components/AnalyticsHeader.react';
+import MainCard from './components/AnalyticsMainCard.react';
```

Comment:
```suggestion
import AnalyticsHeader from 'AnalyticsHeader.react';
import MainCard from 'AnalyticsMainCard.react';
```

No worries, your original codes are correct, but we are able to import the modules just by their name because we are using monorepo :)